.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+require:
+  - rubocop-capybara
```

Comment:
Why do we need this?

Reply:
Could you try removing rubocop-capybara and check if you can still use rubocop properly?

Reply:
apu, i can not detect any offense right now, but when i ran rspec command, i saw this gem in the terminal, few of the offenses were corrected using this gem. 
I have removed it now and ran the rspec command, it works, but since i resolved all the offenses previously, no new offense is detected